src/frontend/src/flows/manage/index.ts

```diff
@@ -115,36 +115,65 @@ const anchorSection = (userNumber: bigint): TemplateResult => html`
 const devicesSection = (
   devices: DeviceData[],
   onAddDevice: (next: "canceled" | "local" | "remote") => void
-): TemplateResult => html`
-    <aside class="l-stack">
-      <div class="t-title t-title--complications">
-        <h2 class="t-title">Added devices</h2>
-        <span class="t-title__complication c-tooltip">
-          <span class="c-tooltip__message c-card c-card--narrow">
-            You can register up to ${MAX_AUTHENTICATORS} authenticator
-            devices (recovery devices excluded)</span>
-            (${numAuthenticators(devices)}/${MAX_AUTHENTICATORS})
-          </span>
-        </span>
-        <button
-          ?disabled=${numAuthenticators(devices) >= MAX_AUTHENTICATORS}
-          class="t-title__complication t-title__complication--end c-tooltip c-tooltip--onDisabled"
-          @click="${async () => onAddDevice(await chooseDeviceAddFlow())}"
-          id="addAdditionalDevice"
-        >
-          <span class="c-tooltip__message c-tooltip__message--right c-card c-card--narrow"
-            >You can register up to ${MAX_AUTHENTICATORS} authenticator devices.
-            Remove a device before you can add a new one.</span
-          >
-          <span class="t-link t-link--discreet">
-            <i class="t-link__icon" aria-hidden=true>+</i>
-            Add new device
+): TemplateResult => {
+  const wrapClasses = ["l-stack"];
+  const isWarning = devices.length < 2;
```

Comment:
nit:

```suggestion
  const shouldWarn = devices.length < 2;
```

I think it's clearer that "this" is not a warning (yet), but instead that we _should_ be warning the user

Reply:
I thought of it as the component has a warning state that is true. So `isWarning` seamed like a good name. Should sounds a bit a like suggestion less then a state the component is in no?